docs/administration/security/authorization.md

```diff
@@ -16,6 +16,263 @@ The remainder of this section will describe how to use the access control policy
 
 <VidStack src="youtube/i859f1WG3Bo" poster="https://img.youtube.com/vi/i859f1WG3Bo/maxresdefault.jpg"/>
 
+## Understanding ACL Concepts
+
+Before diving into specific policies, it's important to understand three fundamental concepts that cause the most confusion:
+
+### The Two-Context Model
+
+Every complete permission set requires rules in BOTH contexts:
+
+1. **Application Context** (`context: application: 'rundeck'`)
+   - Controls access TO projects and system resources
+   - Answers: "Can this user see this project in the list?"
+   - Answers: "Can this user create new projects?"
+   - Required even for project-level users
+
+2. **Project Context** (`context: project: 'ProjectName'`)
+   - Controls actions WITHIN a specific project
+   - Answers: "Can this user run jobs?"
+   - Answers: "Can this user view nodes?"
+
+**Key Point**: A user needs Application-level `read` permission on a project just to see it, PLUS Project-level permissions to do anything inside it.
+
+### Generic vs Specific Resources
+
+This is the most confusing aspect of ACLs. There are two ways to reference resources:
```

Comment:
Can we find a different word to replace "confusing"?  It has a negative connotation - makes me feel like we're admitting poor product experience.  Alternatives i'd consider are:
- intricate
- sophisticated
- nuanced
- syntax-specific

Reply:
removed two instances of "confusing" in the doc.